server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -24,6 +24,7 @@ LEVEL_NUMBER_77 : '77';
 LEVEL_NUMBER_88 : '88';
 
 // keywords
+ABEND : A B E N D;
```

Comment:
Already in CommonLexer.g4, please remove.

We have new design now. All common-keywords goes under  CommonCobolLexer.g4. 
Please check duplication for your newly added keywords from now.  e.g. ABEND and etc.
If they are existing in one of Lexers already, then please remove it. It is inherited by design.